### PR TITLE
blueprint quickstarts

### PR DESCRIPTION
**Pull request by elianerpereira**
Adding the quickstart for Blueprints and their customizations, configurations, and image creation.

### CODE REVIEW

**Comment by elianerpereira** [comment]
I got ACK from all SMEs to merge it. This is now ready to be merged. @Hyperkid123 please, could you help me merge this content?
Thank you.

File: docs/quickstarts/insights-blueprints/insights-creating-blueprint-images.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+metadata:
+  name: create-blueprint
```
Comment (Hyperkid123) [inline docs/quickstarts/insights-blueprints/insights-creating-blueprint-images.yml:2]: The name of files and the name of the resources must match. There is a validation error in the pipeline:
```
19:00:11 #13 3.523 Help topic validation error. Error occured in:  docs/quickstarts/insights-blueprints/insights-blueprint.yaml
19:00:11 #13 3.525 panic: open docs/quickstarts/insights-blueprints/insights-blueprint.yaml: no such file or directory

```

**Review by Hyperkid123 — CHANGES_REQUESTED** [review]

File: docs/quickstarts/insights-blueprints/metadata.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+kind: QuickStarts # kind must always be "QuickStarts"
+name: insights-blueprint
```
Comment (ezr-ondrej) [inline docs/quickstarts/insights-blueprints/metadata.yml:2]: @elianerpereira this name is the important one I think.
```suggestion
name: insights-creating-blueprint-images
```

File: docs/quickstarts/insights-blueprints/insights-creating-blueprint-images.yml [hunk 3]
```diff
@@ -0,0 +1,81 @@
+metadata:
+  name: insights-creating-blueprint-images
+spec:
+  displayName: Creating a blueprint
```
Comment (ezr-ondrej) [inline docs/quickstarts/insights-blueprints/insights-creating-blueprint-images.yml:4]:
```suggestion
  type:
    text: Documentation
    color: orange
  displayName: Creating a blueprint
```

**Comment by elianerpereira** [comment]
Thank you so much, @ezr-ondrej !!

**Comment by ezr-ondrej** [comment]
And validation is green @Hyperkid123 :)

**Comment by ezr-ondrej** [comment]
@elianerpereira can you undraft this PR? :)

**Comment by elianerpereira** [comment]
> @elianerpereira can you undraft this PR? :)

done, thank you :-)

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**